src/sentry/tasks/post_process.py

```diff
@@ -210,6 +211,12 @@ def post_process_group(
                 ):
                     safe_execute(callback, event, futures)
 
+            has_workflow_owners = features.has(
+                "organizations:workflow-owners", project=event.project
```

Comment:
Shouldn't organization features get the organization?

Reply:
Changing this to a project level flag and scoping it to one of my test orgs now. Pretty certain I can't do a team based flag here because there's no actor.